recipes/evalhyd-cli/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.1.0.0" %}
+
+package:
+  name: evalhyd-cli
+  version: {{ version }}
+
+source:
+  url: https://gitlab.irstea.fr/HYCAR-Hydro/evalhyd/evalhyd-cli/-/archive/v{{ version }}/evalhyd-cli-v{{ version }}.tar.gz
+  sha256: 38ebcc4915542b0887d4178724643539355b043a7aa4a68bdeacce4629bd1f11
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.15
+    - ninja
+  host:
+    - xtl >=0.7.5
+    - xtensor >=0.24.6
+    - cli11 >=2.3.1
+    - libevalhyd ==0.1.0
+
+test:
+  commands:
+    - |
+      evalhyd --help
+    - |
+      evalhyd evald --help
+    - |
+      evalhyd evalp --help
```

Comment:
You shouldn't need the `- |` synatx, just regular `-` should suffice.

Reply:
I had issues (on Windows I think) because of the "dash" denoting the YAML sequence, and the "dashes" denoting the help option, but quoting the whole line would not work either, that is why I had it this way. https://github.com/ThibHlln/staged-recipes/commit/4185e1b2e3c0c8c55265fe2a102d7eb019f93199

Reply:
Ok, I have tried replacing `--help` with `-h` to keep it on one line. In my other recipes, I never use double dashes, but single dashes do not seem to trigger problems, so let's see what the CI says.